source/extensions/filters/http/grpc_shim/grpc_shim.cc

```diff
@@ -0,0 +1,170 @@
+#include "extensions/filters/http/grpc_shim/grpc_shim.h"
+
+#include <netinet/in.h>
+
+#include "envoy/common/platform.h"
+
+#include "common/common/enum_to_int.h"
+#include "common/grpc/codec.h"
+#include "common/grpc/common.h"
+#include "common/grpc/status.h"
+#include "common/http/headers.h"
+#include "common/http/utility.h"
+
+#include "extensions/filters/http/well_known_names.h"
+
+#include "absl/strings/str_cat.h"
+
+namespace {
```

Comment:
you can put this inside namespace GrpcShim so you don't need to prefix everything with `Envoy::`